clang/test/dpct/macro_test.cu

```diff
@@ -520,7 +520,7 @@ __global__ void foo8(){
 
 
 //CHECK: #define DFABS(x) (double)sycl::fabs((x))
-//CHECK-NEXT: #define MAX(x, y) sycl::max(x, y)
+//CHECK-NEXT: #define MAX(x, y) dpct::max(a, sycl::sqrt(DFABS(b)))
```

Comment:
Mark this checker, it is a regression

Reply:
And check why using dpct here. Should be sycl?

Reply:
> And check why using dpct here. Should be sycl?

In `math::UsingDpctMinMax`, we check if this API is macro. If it is macro, we use `dpct::max/dpct::min`.

Reply:
> Mark this checker, it is a regression

Done